PyPDF2/_merger.py

```diff
@@ -84,18 +95,38 @@ class PdfMerger:
     :param bool strict: Determines whether user should be warned of all
             problems and also causes some correctable problems to be fatal.
             Defaults to ``False``.
+    :param fileobj: Output file. Can be a filename or any kind of
+            file-like object.
     """
 
     @deprecate_bookmark(bookmarks="outline")
-    def __init__(self, strict: bool = False) -> None:
+    def __init__(self, strict: bool = False, fileobj: StrByteType = "") -> None:
```

Comment:
Should `fileobj` be allowed to be path-like following from #1190?

Additionally, why not make the type `Optional[StrByteType]` and just have it default to None instead of `""`?

Reply:
Good idea to allow Path objects :+1: 

What is the advantage of using `None`?